requirements.txt

```diff
@@ -1,4 +1,21 @@
-numpy
-pennylane>=0.15
-python-dateutil
-requests
+appdirs==1.4.4
+autograd==1.4
+autoray==0.3.2
+cachetools==5.2.0
+certifi==2022.5.18.1
+charset-normalizer==2.0.12
+future==0.18.2
+idna==3.3
+networkx==2.6.0
+ninja==1.10.2.3
+numpy==1.21.6
+PennyLane==0.24.0
+PennyLane-Lightning==0.24.0
+python-dateutil==2.8.2
+requests==2.28.0
+retworkx==0.11.0
+scipy==1.7.3
+semantic-version==2.6.0
```

Comment:
Note: there has been a change to this pinning on `master` of PennyLane, but PennyLane v0.24.0 still requires `semantic-version==2.6.0` (hence a newer version raises an error).